packages/synthetic-chain/package.json

```diff
@@ -28,7 +26,8 @@
   "devDependencies": {
     "@types/better-sqlite3": "^7.6.9",
     "@types/node": "^18.19.14",
-    "ava": "^5.3.1"
+    "ava": "^5.3.1",
+    "tsup": "^8.0.2"
```

Comment:
Btw, doesn't this mean `tsx` is no longer a runtime dependency?

Reply:
great catch. updated